Lesson_Materials/Lecture_03_Managing_Data/index.html

```diff
@@ -280,65 +383,34 @@
 				<!--Slide 16-->
 				<section>
 					<div class="hbox" data-markdown>
-						#### Access modes
```

Comment:
I think it's fair to remove these parts since this will now be a bigger lecture, but we should move them to another section, or perhaps create a new ticket for adding this in somewhere else.

Reply:
We are still mentioning `sycl::no_init` as well as the accessor access modes. These resolution cases are a little bit niche and maybe we could try and fit them into Advanced_Data_Flow? Would be an easy copy-paste and could add to this PR. What do you think?

Reply:
Yeah I agree, let's do that.